### PR TITLE
Refactor Component CSS

### PR DESCRIPTION
**Pull request by kevinNghiem**
Added CSS into Style Tag instead of having it all bunch in the div class
Todo Pages css Refactoring

### CODE REVIEW

File: components/AppButton.vue [hunk 1]
```diff
@@ -1,12 +1,31 @@
...
   </button>
 </template>
 
 <script>
 export default { name: 'AppButton', props: ['data_cy'] }
 </script>
+
+<style >
+.appButtonStyle {
+  @apply shadow;
```
Comment (MarcoGoC) [inline components/AppButton.vue:13]: Good,  just a comment,  you can use a single line for all.

Comment (kevinNghiem) [reply components/AppButton.vue:13]: i tried using a single line but it gave me red marks error but it seem to compile still with the red marks. But i ended up doing it individually .

Comment (MarcoGoC) [reply components/AppButton.vue:13]: I know it is a eslint thing it is annoying but it compiles

Comment (kevinNghiem) [reply components/AppButton.vue:13]: ah okay got it thanks Marco that makes things more easier now

**Pull request merged**